### PR TITLE
Update with new material on basics

### PR DESCRIPTION
**Pull request by semioticrobotic**
This PR suggests adding some new material from [recently published work](https://www.redhat.com/en/blog/sharing-caring-building-clearer-contribution-paths-your-community) by @bproffitt.

### CODE REVIEW

File: onboarding.adoc [hunk 1]
```diff
@@ -46,7 +45,33 @@ No two open source communities are the same.
...
+Quite a few projects have set up tagging on their source code repositories for "first-time contributors" or something similar, which is an excellent way to direct incoming contributors to solving issues and bugs while also introducing newcomers to the source material of the project.
+
+The second question—regarding "where to contribute"—is one that projects don't pose as often as you'd think.
+Project veterans might dismissively say, "what you need is on Git-something-or-other."
+But in which specific repository are the material available?
+Under what organization?
+And what about other, similarly named projects?
+Ensure your project website features clear links to places where new contributors can make an impact on the project. These links shouldn't be buried, either; they should be right on the home page, if possible.
+
+Third is teh question of "how to contribute."
```
Comment (rpaik) [inline onboarding.adoc:68]:
```suggestion
Third is the question of "how to contribute."
```

File: onboarding.adoc [hunk 2]
```diff
@@ -46,7 +45,33 @@ No two open source communities are the same.
 Even for seasoned open source contributors, when they join a new community they will need to get familiar with new tools, new processes, and even new terminologies.
 Depending on the community, seemingly simple terms like contributions, community members, projects, upstream, etc.
 may mean something slightly different and can easily cause confusion for new community members.
-  
+
+== Onboarding basics
+
+As you work to make your project's onboarding process and experience most effective, ask yourself these questions:
+
+- What can contributors add to the project?
```
Comment (rpaik) [inline onboarding.adoc:53]: I feel like there's too much emphasis on `add` here. Someone could contribute to the project by archiving or removing outdated content (and help with house cleaning). Maybe rephrase to something like "What can community members contribute to the project" (and something similar to the rest of the bullets)?

File: onboarding.adoc [hunk 3]
```diff
@@ -46,7 +45,33 @@ No two open source communities are the same.
...
+- Where do contributors add to the project?
+- How do contributors add to the project?
+
+Answering the first question about your project's onboarding process requires a good deal of work work.
+Incoming contributors need to know what would be the most helpful contributions they could make.
+Quite a few projects have set up tagging on their source code repositories for "first-time contributors" or something similar, which is an excellent way to direct incoming contributors to solving issues and bugs while also introducing newcomers to the source material of the project.
+
+The second question—regarding "where to contribute"—is one that projects don't pose as often as you'd think.
```
Comment (rpaik) [inline onboarding.adoc:61]: It sounds like there's too much emphasis on code (esp. with `Git-something-or-other`). Rather than only mentioning repos, it'd be good to list other areas (e.g. forums, meetups, wiki's, etc.) where people can contribute.

File: onboarding.adoc (hunk 1 above)
Comment (semioticrobotic) [reply onboarding.adoc:68]: Gah! Silly mistake. Thanks for catching, @rpaik.

File: onboarding.adoc (hunk 2 above)
Comment (semioticrobotic) [reply onboarding.adoc:53]: Great point. Let me see what I can do.

Comment (semioticrobotic) [reply onboarding.adoc]: What do you think of this?

File: onboarding.adoc (hunk 3 above)
Comment (semioticrobotic) [reply onboarding.adoc:61]: Tried a revision. Thoughts?

Comment (rpaik) [reply onboarding.adoc:61]: 👍

File: onboarding.adoc (hunk 2 above)
Comment (rpaik) [reply onboarding.adoc]: 👍

**Review by rpaik — APPROVED** [review]

**Review by rpaik — APPROVED** [review]

**Comment by semioticrobotic** [comment]
Thanks, @rpaik!

**Review by quaid — APPROVED** [review]
Look quite good, I'm approving to merge, then will follow up with a merge that is for the Z stream i.e. 2.0.1
